projects/cdk/tokens/active-element.ts

```diff
@@ -33,8 +33,14 @@ export const TUI_ACTIVE_ELEMENT = new InjectionToken<Observable<EventTarget | nu
                 focusout$.pipe(
                     takeUntil(mousedown$),
                     repeatWhen(() => mouseup$),
-                    withLatestFrom(skip$),
-                    filter(([{target}, element]) => target !== element),
+                    withLatestFrom(removedElement$),
+                    filter(([{target}, removedElement]) => {
+                        if (removedElement) {
```

Comment:
No need for `if`, can be just `!removedElement || removedElement.contains(target as Node)`